src/output-view.js

```diff
@@ -0,0 +1,109 @@
+import {
+    Transformime,
+    TextTransformer,
+    ImageTransformer,
+    HTMLTransformer
+} from "transformime";
+
+import {
+    consoleTextTransform,
+    markdownTransform,
+    LaTeXTransform,
+    PDFTransform
+} from "transformime-jupyter-transformers";
+
+/**
+ * Displays output area state.
+ */
+export class OutputView {
+    
+    /**
+     * Public constructor
+     * @param  {IOutputModel} model    output model that this view represents
+     * @param  {Document}     document handle to the Document instance that 
+     *                                 the output will be rendered in.
+     */
+    constructor(model, document) {
+        this.model = model;
+        
+        let el = document.createElement('div');
+        Object.defineProperty(this, 'document', {get: () => document});
+        Object.defineProperty(this, 'el', {get: () => el});
+        
+        // Transformers are in reverse priority order
+        // so that new ones can be `push`ed on with higher priority
+        let transformers = [
+            TextTransformer,
+            PDFTransform,
+            ImageTransformer,
+            // SVG would go here, IF I HAD ONE
+            consoleTextTransform,
+            LaTeXTransform,
+            markdownTransform,
+            HTMLTransformer
+            // JavaScript would go here, IF I HAD ONE
+        ];
+        this.transformime = new Transformime(transformers);
+        
+        this._bindEvents();
+    }
+    
+    /**
+     * Listen to relevant model events.
+     */
+    _bindEvents() {
+        this.model.on('change', this._modelChange.bind(this));
+    }
+    
+    /**
+     * Handle when the model changes.
+     * @param {Output[]} newState new state of the model
+     * @param {Output[]} oldState old, previous, state of the model
+     */
+    _modelChange(newState, oldState) {
+        // TODO: Diff states.
+        // Fast, but ugly way to clear all of the elements.
+        while (this.el.firstChild) {
+            this.el.removeChild(this.el.firstChild);
+        }
+        
+        // Use transformime to render state.
+        let orderPromise = Promise.resolve();
```

Comment:
Would be cool if you commented that you're building a promise chain so that the series of newStates get output in the right order.